rust_src/src/editfns.rs

```diff
@@ -1113,7 +1113,7 @@ pub fn buffer_substring_no_properties(mut beg: LispObject, mut end: LispObject)
 // offload some work from GC.
 #[no_mangle]
 pub extern "C" fn save_excursion_save() -> LispObject {
-    let window = selected_window().as_window_or_error();
+    let window: LispWindowRef = selected_window().into();
```

Comment:
Are we approaching the time for `selected_window` to return a `LispWindowRef`?

Reply:
Maybe. There are a few uses of `selected_window().as_window()`, and I don't understand what guarantees are made about the selected window.

Reply:
`Option<LispWindowRef>` is likely the right thing for it to return. This allows for a nil selection initially.

Reply:
But I think it is almost always `Some`.